### PR TITLE
UMIP - WBTC as collateral created

### PR DESCRIPTION
**Pull request by OpenDAO-Logan**

### CODE REVIEW

File: UMIPs/umip-draft_wbtc_collateral.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Headers
+| UMIP-#    |                                                                                                                                          |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| UMIP Title | Add Wrapped Bitcoin as a collateral currency              |
+| Authors    | Logan (logan@opendao.io) |
+| Status     |Draft                                                                                                                                   |
+| Created    | Feb 2, 2021                                                                                                                           |
```
Comment (smb2796) [inline UMIPs/umip-draft_wbtc_collateral.md:7]: Can you create an UMA Discourse discussion introducing this UMIP and add another row and add another row linking to that discussion? https://discourse.umaproject.org/

File: UMIPs/umip-draft_wbtc_collateral.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+ Additionally, the conversion bewteen BTC/WBTC and WBTC/BTC is completed in a fast and transparent manner, thereby increasing 'good faith' in the collateral. Users may refer to WBTC’s “orderbook dashboard” (https://WBTC.network/dashboard/order-book) to see a record of all minting and burning of WBTC on the Ethereum network. Also, users can access WBTC’s “proof of assets dashboard” (https://WBTC.network/dashboard/audit) to view the transaction amounts and the total amounts of BTC held under custody.
+
+
+For final consideration, WBTC is centralized in that the BTC collateral is custodied by BitGo and thus presents a failure-bottleneck; however, it is important to note that BitGo is one of the most reputable centralized custodians in the world and has highly advanced coin security protocols in place. Furthermore, BitGo is covered by digital asset insurance worth up to $100 million in the worst case scenario.
```
Comment (Tommy1231232) [inline UMIPs/umip-draft_wbtc_collateral.md:48]: Thanks for calling this out!

File: UMIPs/umip-draft_wbtc_collateral.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+The motivation for adding WBTC as a supported collateral in the global whitelist contract is to add utility to the UMA architecture; access to WBTC as a collateral currency creates opportunities for partner projects to better leverage the protocol.
+
+Wrapped Bitcoin (WBTC) is an ERC-20 token on the Ethereum blockchain that represents Bitcoin. Each WBTC is backed 1:1 with Bitcoin. Wrapped Bitcoin allows for Bitcoin transfers to be conducted quicker on the Ethereum blockchain and opens up the possibility for BTC to be used in the decentralized finance ecosystem. The Bitcoin collateral is held in custody by the centralized custodian, BitGo. Bitcoin can be converted to Wrapped Bitcoin and vice versa quite easily.
+
+At the time of writing, WBTC is trading at $33,767.47 with a 24-hour trading volume of $346,285,873. There is a circulating supply of 116,870 WBTC coins with a max supply of 116,870. Uniswap is the current most active market trading it.
+
+​
+## Technical Specification
+To accomplish this upgrade, two changes need to be made:
+- The WBTC address, 0x2260fac5e5542a773aa44fbcfedf7c193bc2c599](https://etherscan.io/token/0x2260fac5e5542a773aa44fbcfedf7c193bc2c599),
```
Comment (smb2796) [inline UMIPs/umip-draft_wbtc_collateral.md:24]:
```suggestion
- The WBTC address, [0x2260fac5e5542a773aa44fbcfedf7c193bc2c599](https://etherscan.io/token/0x2260fac5e5542a773aa44fbcfedf7c193bc2c599), 
```

**Review by smb2796 — APPROVED** [review]
Going to merge in and update the title/status

**Pull request merged**